alainmateso/iFind
# Ft add post image

#### What does this PR do?
Add functionality to add an image when posting a new item

#### Description of Task to be completed?

- Give a user the ability to upload an image when posting a new item
- Verify if the file is a proper image file
- Upload the image to cloudinary
- Return and store the image URL

#### How should this be manually tested?
1. Clone the Github repository
2. Navigate to ```ft-add-post-image-#168697726``` branch
3. Make sure you have a cloudinary account and add the following Environment Variables to your .env file:
`CLOUD_NAME=xxxxxx`
`API_KEY=xxxxxxx`
`API_SECRET=xxxxxxx `

4. Run npm install to install development dependencies by running ```npm install```
5. Run ```npm run dev```
6. Fill all necessary fields and choose an image file
7. send POST request to api/v1/posts

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168697726
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
